packages/components/src/popover/index.js

```diff
@@ -95,6 +96,8 @@ class Popover extends Component {
 	}
 
 	toggleAutoRefresh( isActive ) {
+		const { clearInterval, setInterval } = this.props;
```

Comment:
It begs the question: If the component is taking advantage of `withSafeInterval`, does it really need to call `clearInterval` itself? As I can tell, the `clearInterval` would only be called as a result of `componentWillUnmount`, the same which would be handled by `withSafeIntervals`.

Reply:
Hm, I'm not following. The existing code calls `clearInterval` [here](https://github.com/WordPress/gutenberg/pull/14818/files#diff-ccdf667da5123277e79748286c32fda7R118). So I'd say, yes, there is still the need to stop the callback from executing. Or did I miss something?

Reply:
The purpose of a `withSafeInterval` is that it handles this on behalf of the component:

https://github.com/WordPress/gutenberg/pull/14818/files#diff-fe5750aa4c4751d70bcf38cb66604118R35

The existing code didn't use `withSafeInterval`, so it needed to call it itself.